trlx/trainer/nn/ppo_models.py

```diff
@@ -124,6 +124,7 @@ class PPOConfig(MethodConfig):
     ref_std: Optional[float]
     cliprange_reward: float
     gen_kwargs: dict
+    gen_inference_kwargs: Optional[dict] = None
```

Comment:
Double-checking: In which cases will `gen_kwargs` and `gen_inference_kwargs` be different? (Just don't want to duplicate fields containing the same info)

Reply:
In some cases, we need sampling for the exploration phase and beam search or something else for the inference that is why I want to add this one like RM4LMs config. Is it reasonable?

Reply:
e.g. https://github.com/allenai/RL4LMs/blob/main/scripts/training/task_configs/narrative_qa/t5_ppo_on_supervised.yml

Reply:
by default, if people do not settings this argument it will be the same as `gen_kargs`.

Reply:
Ah okay. So one setting is for evaluation/inference and the other for the actual in-training policy. Thanks! We should also add a docstring to the `PPOConfig` below the `gen_kwargs` 😅  https://github.com/CarperAI/trlx/blob/220c8f3a3c91738f26e8522effb90b5caa119df4/trlx/trainer/nn/ppo_models.py#L107-L108

Reply:
Maybe it would be a little clearer to distinct between the two with some renaming. For example if `gen_kwargs` are kept for evaluation, but new kwargs can be called `gen_experience_kwargs` if beam search/different options are needed specifically for PPO exploration

Reply:
That's a cool idea. I can change it but just make sure we have some notes for users they also need to change their configure then.